source/io/module_propertyNode.js

```diff
@@ -0,0 +1,150 @@
+// Using a modified UMD module format. Specifically a modified returnExports (no dependencies) version
+(function (root, globalName, factory) {
+    'use strict';
+    var buildGlobalNamespace = function () {
+        var buildArgs = Array.prototype.slice.call(arguments);
+        return globalName.split('.').reduce(function (currObj, subNamespace, currentIndex, globalNameParts) {
+            var nextValue = currentIndex === globalNameParts.length - 1 ? factory.apply(undefined, buildArgs) : {};
+            return currObj[subNamespace] === undefined ? (currObj[subNamespace] = nextValue) : currObj[subNamespace];
+        }, root);
+    };
+
+    if (typeof define === 'function' && define.amd) {
+        // AMD. Register as a named module.
+        define(globalName, [], factory);
+    } else if (typeof module === 'object' && module.exports) {
+        // Node. "CommonJS-like" for environments like Node but not strict CommonJS
+        module.exports = factory();
+    } else {
+        // Browser globals (root is window)
+        buildGlobalNamespace();
+    }
+}(this, 'NationalInstruments.Vireo.ModuleBuilders.assignPropertyNode', function () {
+    'use strict';
+
+    /* global Map */
+
+    // Static Private Variables (all vireo instances)
+    var ERRORS = {
+        // Shared
+        NO_ERROR: {
+            CODE: 0,
+            MESSAGE: ''
+        },
+
+        kNIObjectReferenceIsInvalid: {
+            CODE: 1055,
+            MESSAGE: 'Object reference is invalid.'
+        }
+    };
+
+    var formatMessageWithException = function (messageText, exception) {
```

Comment:
This static function is duplicated in module_httpClient and module_javaScriptInvoke.

Please move it to module_coreHelpers and remove the duplicates